### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dreamviewer12**
Add a missing info. for running the project.

### CODE REVIEW

**Review by iamblizzard — APPROVED** [review]

**Review by iamblizzard — COMMENTED** [review]
Need to be done.

**Review by iamblizzard — COMMENTED** [review]
Need to be done.

File: README.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ Commands 2-5 should be run only to reset the project.
 3. Delete the requests.pickle file.
 4. Run remaster.py
 5. Run temp.py
-6. Run ani.py in a separate cmd to visualize.
+6. Run ani.py in a separate cmd to visualize(Note: It can only be executed on Python 2.x).
```
Comment (shreybatra) [inline README.md:62]: Not necessarily. ```can be executed on python 2.7 and above.```

**Review by shreybatra — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (dreamviewer12) [reply README.md:62]: Done @shreybatra

Comment (shreybatra) [reply README.md:62]: @dreamviewer12 can't see the changes. It's still the same.

**Review by shreybatra — CHANGES_REQUESTED** [review]

**Pull request closed without merging**